src/js/dataMapping/search/filterFields.js

```diff
@@ -107,5 +110,8 @@ export const accountAwardsFields = {
     cfdaTitle: 'award__transaction__assistance_data__cfda__program_title',
     naics: 'award__transaction__contract_data__naics',
     naicsDescription: 'award__transaction__contract_data__naics_description',
-    psc: 'award__transaction__contract_data__product_or_service_code'
+    psc: 'award__transaction__contract_data__product_or_service_code',
+    pricingType: 'award__latest_transaction__contract_data__type_of_contract_pricing',
+    setAside: 'award__latest_transaction__contract_data__type_set_aside',
+    extentCompeted: 'award__latest_transaction__contract_data__extent_competed'
```

Comment:
i believe these are messed up (my fault).  Change from latest_trasaction to transaction
Should be: 'award__transaction__contract_data__type_set_aside'
not: 'award__latest_transaction__contract_data__type_set_aside'